src/main/java/packman/service/aloneList/AloneListPackService.java

```diff
@@ -3,9 +3,50 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
+import packman.dto.list.ListResponseMapping;
+import packman.dto.pack.PackCreateDto;
+import packman.entity.Category;
+import packman.entity.Pack;
+import packman.entity.packingList.PackingList;
+import packman.repository.CategoryRepository;
+import packman.repository.FolderPackingListRepository;
+import packman.repository.UserRepository;
+import packman.repository.packingList.AlonePackingListRepository;
+import packman.repository.packingList.PackingListRepository;
+
+import static packman.validator.IdValidator.*;
+import static packman.validator.LengthValidator.validatePackLength;
+import static packman.validator.Validator.validateListCategory;
+import static packman.validator.Validator.validateUserAloneList;
 
 @Service
 @Transactional
 @RequiredArgsConstructor
 public class AloneListPackService {
+    private final UserRepository userRepository;
+    private final FolderPackingListRepository folderPackingListRepository;
+    private final CategoryRepository categoryRepository;
+    private final PackingListRepository packingListRepository;
+    private final AlonePackingListRepository alonePackingListRepository;
+
+    public ListResponseMapping createPack(PackCreateDto packCreateDto, Long userId) {
+        Long aloneId = Long.valueOf(packCreateDto.getListId());
+        Long categoryId = Long.valueOf(packCreateDto.getCategoryId());
+        String packName = packCreateDto.getName();
+
+        validateUserId(userRepository, userId);
+
+        PackingList packingList = validatePackingListId(packingListRepository, aloneId);
+        validateUserAloneList(userId, validateAlonePackingListId(alonePackingListRepository, aloneId));
+
+        Category category = validateCategoryId(categoryRepository, categoryId);
+
+        validatePackLength(packName);
+        validateListCategory(aloneId, category);
```

Comment:
```suggestion
        validateListCategory(packingList, category);
```
이러면 join 더 줄일 수 있을 것 같아욥!

Reply:
`validateListCategory`부분이 따로 jpa메서드를 써서 한 게 아니라서 확인해보니까 sql문 없이 작동하더라고! 고냥 안에 타고타고 가서 내용이 같은지만 확인하는것 같아. 나는 packingList전체를 비교하는 것보다 id비교하는게 더 빠를 것 같아서 한건데 어떻게 생각행?